src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -357,7 +357,11 @@ static int lsf_driver_submit_shell_job(lsf_driver_type *driver,
                                        const char *submit_cmd, int num_cpu,
                                        const char *run_path) {
     int job_id;
-    char *tmp_file = (char *)util_alloc_tmp_file("/tmp", "enkf-submit", true);
+    constexpr int OUTPUT_FILE_SIZE = 32;
+    char tmp_file[OUTPUT_FILE_SIZE];
+    strncpy(tmp_file, "/tmp/enkf-submit-XXXXXXXXXX", OUTPUT_FILE_SIZE);
```

Comment:
why so many X's? `man 3 mkstemp` says there should be 6.

Reply:
Changed it to six